Tools/autotest/common.py

```diff
@@ -1626,11 +1615,14 @@ def test_arm_feature(self):
         # TEST ARMING/DISARM
         self.set_parameter("ARMING_RUDDER", 2)  # allow arm and disarm with rudder on first tests
         interlock_channel = 8  # Plane got flighmode_ch on channel 8
-        if self.mav.mav_type is not mavutil.mavlink.MAV_TYPE_HELICOPTER:  # heli don't need interlock option
+        if not self.is_heli():  # heli don't need interlock option
```

Comment:
s/don't need/needs/

Reply:
the comment is right, heli don't need to set explicitly interlock, as it is by default on channel 8